### PR TITLE
Fix/refactor

### PR DESCRIPTION
**Pull request by darlaevans2000**
## Is this a fix or a feature?
fix
## What is the change?
just refactors code
## What does it fix?
fixes sloppy code
## Where should the reviewer start?
main.js
## How should this be tested?
open index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h6 class="filter-label">Filter Starred Ideas</h6>
         <button id="searchButton" class="magnification">
           <img src="./assets/search.svg" alt="magnifying glass">
         </button>
-        <input id="searchInput" value="Search ideas" class="search-ideas"></input>
+        <input id="searchInput" placeholder="Search ideas" class="search-ideas"></input>
       </div>
```
Comment (BigBike96) [inline index.html:34]: Nice change to improve the functionality!

File: src/main.js [hunk 2]
```diff
@@ -9,24 +8,20 @@ var starImage = document.querySelectorAll('.star');
...
 
 topForm.addEventListener('input', checkInputs);
 window.addEventListener('click', clickHandler);
-
 searchInput.addEventListener('input', searchIdeas);
-searchInput.addEventListener('keyup', clearSearch);
-//Functions
+searchButton.addEventListener('click', clearSearch);
+
 function clickHandler(event) {
```
Comment (BigBike96) [inline src/main.js:25]: This is a change that Kristi recommended. Seems to make sense

**Review by rhen92 — APPROVED** [review]
Good work everyone!!!

File: src/main.js [hunk 3]
```diff
@@ -170,7 +167,7 @@ function searchIdeas(event) {
   } else {
     var searchedIdeas = [];
     for (var i = 0; i < ideaCards.length; i++) {
-      if (ideaCards[i].body.includes(searchInput.value) || ideaCards[i].title.includes(searchInput.value)) {
+      if (ideaCards[i].body.includes(searchInput.value.toUpperCase()) || ideaCards[i].title.includes(searchInput.value.toUpperCase()) || ideaCards[i].body.includes(searchInput.value.toLowerCase()) || ideaCards[i].title.includes(searchInput.value.toLowerCase())) {
```
Comment (BigBike96) [inline src/main.js:170]: This change fixes a bug that only searched for one letter on user input

**Review by BigBike96 — APPROVED** [review]
Sweet! Complete

**Pull request merged**